storage/src/main/java/com/brandwatch/ahmedanalytics/storage/Config.java

```diff
@@ -0,0 +1,32 @@
+package com.brandwatch.ahmedanalytics.storage;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+import java.util.Properties;
+
+@Configuration
+@EnableScheduling
```

Comment:
Does `@EnableScheduling` need to be here? I can't see any scheduled methods.

Reply:
No it doesn't. I forgot to remove it 😅